kinto/tests/test_config.py

```diff
@@ -0,0 +1,25 @@
+from ..config import render_template as origin
+from .support import unittest
+
+class ConfigTest(unittest.TestCase):
+
+#to check if values dict is not empty
+    def test_values_is_not_empty(self):
+        self.assertNotEqual(origin.values, {})
```

Comment:
I am trying to see if values has elements. If it has elements (is not empty) then it is correct- as it will have elements only if it has passed through the if statements and init function. 
If it is empty (has no elements), there was a failure somewhere.